packages/opencensus-instrumentation-grpc/src/grpc.ts

```diff
@@ -14,12 +14,16 @@
  * limitations under the License.
  */
 
-import {BasePlugin, CanonicalCode, PluginInternalFiles, RootSpan, Span, SpanContext, SpanKind, TraceOptions} from '@opencensus/core';
+import {BasePlugin, CanonicalCode, PluginInternalFiles, RootSpan, Span, SpanContext, SpanKind, TagMap, TraceOptions} from '@opencensus/core';
 import {deserializeSpanContext, serializeSpanContext} from '@opencensus/propagation-binaryformat';
 import {EventEmitter} from 'events';
 import * as grpcTypes from 'grpc';
 import * as lodash from 'lodash';
 import * as shimmer from 'shimmer';
+import * as clientMetrics from './grpc-stats/client-metrics';
+import * as serverMetrics from './grpc-stats/server-metrics';
+
+const sizeof = require('object-sizeof');
```

Comment:
Is this possible to do as a TypeScript `import`?

Reply:
```object-sizeof``` package is missing typings. I tried to use https://github.com/miktam/sizeof/blob/master/index.d.ts but didn;t work.

Reply:
OK, makes sense.